src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,63 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
```

Comment:
Updated :+1:

May be worth it to update `security.console.user_password_encoder_command` to `security.command.user_password_encoder`